agent/src/main.rs

```diff
@@ -236,7 +236,8 @@ fn init_router(state: &Arc<RunnerState>) -> Router {
 
     let mut router: Router<Arc<RunnerState>, Body> = Router::new()
         .route("/monitor/ping", get(ping))
-        .route("/monitor/status", get(monitor_status));
+        .route("/monitor/status", get(monitor_status))
+        .route("/meta", get(meta));
```

Comment:
Can we stick this under something like `/meta/hostinfo`? I want the /meta/ namespace to be open to future expansion.

Reply:
Thx, great input! Will add the very easy stats details to my PR and create separate issues for those that are more complex.